automation/taskcluster/decision_task.py

```diff
@@ -137,6 +137,43 @@ def release(channel, is_staging, version_name):
     return (build_tasks, signing_tasks, push_tasks)
 
 
+def nightly_to_production_app(is_staging, version_name):
+    # Since the Fenix nightly was launched, we've pushed it to the production app "org.mozilla.fenix" on the
+    # "nightly" track. We're moving towards having each channel be published to its own app, but we need to
+    # keep updating this "backwards-compatible" nightly for a while yet
+    channel = 'nightly'
+    variants = get_variants_for_build_type(channel)
```

Comment:
We'll want to use the production build type/variant for production, simply in order to get the right environment in Sentry crash reports.

Reply:
If we do a `nightly` build and put it on the `nightly` track, we want it to be associated with `nightly` on Sentry, right?
We don't want to pollute the `production` environment in Sentry with `nightly` crashes - I might be misunderstanding our Sentry setup :thinking: